.github/ISSUE_TEMPLATE/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- (REQUIRED) What is the nature of this issue? -->
+## This Community doc request is a:
+- [ ] New topic request
+- [ ] Topic content updates
+- [ ] Other
+
+<!-- (REQUIRED) What is the Community issue and code PR? -->
+## Community GitHub issue link:
+
+## Community GitHub code PR link:
+
+<!-- (REQUIRED) What is the Magento Open Source/Commerce and version for the PR? Will it be back or forward ported? -->
+## Magento version:
+- [ ] Open Source
+- [ ] Commerce
+
+## Back and Forward porting:
+
+<!-- (REQUIRED) What new information or updates are required for your Community contribution? -->
+## Content for Community contribution:
+
+<!-- (OPTIONAL) Where would you expect this content in DevDocs/Merchant guides? -->
+## Possible location:
```

Comment:
I think we should remove this. We already know that most people struggle with where to put things on devdocs.

Reply:
Can definitely remove and agree.

Reply:
New features agree. Targeted updates, no.